src/ukify/ukify.py

```diff
@@ -1395,9 +1398,12 @@ def config_example(self) -> tuple[Optional[str], Optional[str], Optional[str]]:
 def apply_config(namespace, filename=None):
     if filename is None:
         filename = namespace.config
-    if filename is None:
+        # If config file is specified, check if it exists
+        if filename is not None and not os.path.isfile(filename):
```

Comment:
This would disallow symlinks. Please use the same style as in `60-ukify.install.in`.

Reply:
I can do that and just try reading all the configs one by one, but I've tested it and seems like that os.path.isfile returns true when it's either file or symlink anyways so opinions?

Reply:
Indeed.